library/core/src/panic/unwind_safe.rs

```diff
@@ -248,6 +248,7 @@ impl RefUnwindSafe for crate::sync::atomic::AtomicBool {}
 impl<T> RefUnwindSafe for crate::sync::atomic::AtomicPtr<T> {}
 
 #[stable(feature = "catch_unwind", since = "1.9.0")]
+#[rustc_const_unstable(feature = "const_deref", issue = "none")]
 impl<T> Deref for AssertUnwindSafe<T> {
```

Comment:
These are never constructed in const contexts, so with the same reasoning as for String/Vec, we should not add const methods or impls on them.